firmware/applications/SignBuddy/sensors.c

```diff
@@ -23,11 +24,23 @@ static state_t state;
 int get_flex_data(flex_data_t *data)
 {
   for (int i = 0; i < FLEX_SENSOR_CNT; i++) {
-    data->sensor_data[i] = rand() % 150;
+    data->sensor_data[i] = adc_read();
   }
   return RET_OK;
 }
 
+void tx_flex_data(flex_data_t *data)
+{
+  for (int i = 0; i < FLEX_SENSOR_CNT; i++) {
+    uint8_t tx_data[2] = { data->sensor_data[i] & 0xff, data->sensor_data[i] >> 8 };
+    for (int j = 0; j < 2; j++) {
+      ble_uart_tx(tx_data[j]);
+      uint32_t start_time = system_time_get();
+      while (system_time_cmp_ms(start_time, system_time_get()) < 100);
```

Comment:
Will modify `ble_uart_tx()` to be able to handle input other than uint8_t

Reply:
i think you should change it to `uint8_t *data, uint32_t length` if anything, and encoding should be handled in a higher level module. not for this pr or anything, but just in the future

Reply:
Yup sounds good, we can make that change when we integrate ble_uart with the logger